internal/pgengine/sql/ddl.sql

```diff
@@ -187,15 +188,15 @@ BEGIN
     PERFORM 1
         FROM timetable.active_session s
         WHERE
-            s.client_pid <> worker_pid
+            s.client_addr <> inet_client_addr()
```

Comment:
What if we run several instances on the same host? Then inet client addr will be the same

Reply:
s.client_addr <> inet_client_addr()
AND s.client_name = worker_name   # !!!!

If there are multiple instances on the same host, each clientname must be unique.

Reply:
The client name must be absolutely unique, even if your workers are running on different hosts.

Reply:
And that, in turn, means we have a bug that must be fixed immediately. 🤔